SentinelOne/cloud_funnel2.0/ingest/parser.yml

```
@@ -0,0 +1,661 @@
+name: cloudfunnel2.0
+pipeline:
+  - name: json_event
+    external:
+      name: json.parse-json
+      properties:
+        input_field: "{{original.message}}"
+
+  # Time conversions
+  - name: convert_timestamp
+    filter: '{{json_event.message.get("@timestamp") != None}}'
+    external:
+      name: date.parse
+      properties:
+        input_field: '{{json_event.message["@timestamp"]}}'
+        output_field: start_date
+        format: timestamp
+
+  - name: convert_process_exec_start
+    filter: "{{json_event.message.event.source != null and json_event.message.event.source.executable.creationTime.millisecondsSinceEpoch != null and json_event.message.event.source.executable.creationTime.millisecondsSinceEpoch | length == 13}}"
+    external:
+      name: date.parse
+      properties:
+        input_field: "{{json_event.message.event.source.executable.creationTime.millisecondsSinceEpoch[:-3]}}.{{json_event.message.event.source.executable.creationTime.millisecondsSinceEpoch[-3:]}}"
```

`date.parse` now supports milliseconds timestamp
```suggestion
    filter: "{{json_event.message.event.source != null and json_event.message.event.source.executable.creationTime.millisecondsSinceEpoch != null}}"
    external:
      name: date.parse
      properties:
        input_field: "{{json_event.message.event.source.executable.creationTime.millisecondsSinceEpoch}}"
```